home/management/commands/clear_svg_to_png_map.py

```diff
@@ -0,0 +1,9 @@
+from django.core.management.base import BaseCommand
+
+from home.models import SVGToPNGMap
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        SVGToPNGMap.objects.all().delete()
```

Comment:
This script leaves the PNG files on the filesystem. If we're going to provide a handy script, I think it ought to be thorough. Please make sure that the changes to the database are committed before removing files from the filesystem.

Reply:
done